File: Go/FileControl.go
package main

import (
	"fmt"
	"log"
)
import "io/ioutil"

type FileControl struct {}

func (fileControl FileControl) execute(num int) {
	switch num {
	case 1: fileControl.question1()
	case 2: fileControl.question2()
	case 3: fileControl.question3()
	case 4: fileControl.question4()
	}
}

func (fileControl FileControl) question1() {
	outputString := []byte("This is the string to write.\n")
	err := ioutil.WriteFile("output.txt", outputString, 0777)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Println("Completed.")
}

func (fileControl FileControl) question2() {
	panic("implement me")
}

func (fileControl FileControl) question3() {
	panic("implement me")
}

func (fileControl FileControl) question4() {
	panic("implement me")
}

File: Dart/lib/FileControl.dart
import 'dart:io';
import 'package:Dart/Executable.dart';

class FileControl extends Executable {
  @override
  void Question1() {
    File file = File('output.txt');
    file.writeAsString("This is the string to output.");
  }

  @override
  void Question2() {
    // TODO: implement Question2
  }

  @override
  void Question3() {
    // TODO: implement Question3
  }

  @override
  void Question4() {
    // TODO: implement Question4
  }

}